lib/fake_braintree/sinatra_app.rb

```diff
@@ -185,6 +185,14 @@ def hash_from_request_body_with_key(key)
       CreditCard.new(updates, options).update
     end
 
+    # Braintree::PaymentMethod.delete
+    delete '/merchants/:merchant_id/payment_methods/any/:credit_card_token' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.